### ISSUE TITLE
cannot pass more than 3 parameters

### ISSUE THREAD

**Issue by kavya05** [ojacques/hubot-apiai#12]
hey i am not able to pass more than 3 parameters, its it related to coffee script ?

**Comment by ojacques** [comment]
@kavya05 - this is strange, although I never used more than 3 params myself.

[this line](https://github.com/ojacques/hubot-apiai/blob/master/src/apiai.coffee#L68) does not restrict the number of parameters. 

Maybe a limitation from [`apiai`](https://www.npmjs.com/package/apiai) ?

**Comment by kavya05** [comment]
@ojacques yes maybe its a problem with apiai, im still not able to pass more than 3 parameters.

**Comment by ADAM-M-C** [comment]
Just tested. It does work with more than 3 params

**Comment by kavya05** [comment]
@ACBiggTasty have you tried passing parameters for two different intents? like first 3 parameters for intent A and rest of the 2 parameters for intent B, if the total parameters was 5.

**Comment by ADAM-M-C** [comment]
![image](https://user-images.githubusercontent.com/37308921/37346574-0c049e68-26a6-11e8-95d4-7c0dc198810d.png)

![image](https://user-images.githubusercontent.com/37308921/37346637-33842d46-26a6-11e8-849a-3874095d010a.png)
![image](https://user-images.githubusercontent.com/37308921/37346705-668321e8-26a6-11e8-9408-a5ba4dcaf60f.png)


![image](https://user-images.githubusercontent.com/37308921/37346536-efd21856-26a5-11e8-879e-72aca61f5d97.png)

Looks like it works for me. are you trying to pass two intents to the same function simultaneously?

**Comment by kavya05** [comment]
@ACBiggTasty  can you show me your `apiai.coffee` code just want to see how are you emitting those parameters, i guess i have messed up with white spaces in the code, so im not able to pass more params
